### DISCUSSION TITLE
no show contributions

### DISCUSSION THREAD

**Discussion by muhammadkodir-alijonov** [community/community#78194] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello everybody

I work in Visual Studio, when I commit the changes in my code and push them to Git Hub, the "contributions" table does not appear green. it didn't happen.
I need your help to find a solution

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @mukhammadkodir-alijonov ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!

**Comment by Kiran1689** [comment]
@mukhammadkodir-alijonov  Make sure that your commits are linked with the correct email address that is linked to your GitHub account (The email you used while creating your github account) . GitHub uses that email address for counting your contributions.

Please run this command and set your email address

`git config user.email "your.email@example.com"
`

Replace "your.email@example.com" with your actual email address

After running this command, try to commit and push, Now your contributions should reflect on your GitHub profile

**Reply by Kiran1689** [reply]
Hi @mukhammadkodir-alijonov If the issue is solved please make it as answered.

**Comment by muhammadkodir-alijonov** [comment]
hi
I solved it
I used the git init command, which I deleted without knowing, and then I
solved the problem using that

**Reply by Kiran1689** [reply]
Cool 👍

**Comment by Kabilduke** [comment]
Make sure you are the actual author of the commits. If you're working in a team or collaborating on a project, confirm that the commits are authored by you.
Use the following command to see the author of the last commit:
`git log -1`

Sometimes, browser cache issues can affect the display of contributions. Try clearing your browser cache or viewing the contributions on a different browser.
